Tests/SwiftParserTest/TrailingCommaTests.swift

```diff
@@ -15,103 +15,62 @@ import SwiftSyntax
-      fixedSource: "(1, 2, 3,)",
-      experimentalFeatures: .trailingComma
+      fixedSource: "(<#expression#>,)"
     )
   }
 
   func testArgumentList() {
     assertParse("f(1, 2, 3,)", experimentalFeatures: .trailingComma)
 
-    assertParse(
-      "fℹ️(1, 2, 3,1️⃣",
-      diagnostics: [
-        DiagnosticSpec(
-          message: "expected ')' to end function call",
-          notes: [NoteSpec(message: "to match this opening '('")],
-          fixIts: ["insert ')'"]
-        )
-      ],
-      fixedSource: "f(1, 2, 3,)",
-      experimentalFeatures: [.trailingComma]
-    )
-
     assertParse(
       "f(1️⃣,)",
       diagnostics: [DiagnosticSpec(message: "expected value in function call", fixIts: ["insert value"])],
-      fixedSource: "f(<#expression#>,)",
-      experimentalFeatures: .trailingComma
+      fixedSource: "f(<#expression#>,)"
     )
   }
 
-  func assertIf(
-    experimentalFeatures: Parser.ExperimentalFeatures? = nil,
-    _ source: () -> String,
-    body: () -> String,
-    file: StaticString = #file,
-    line: UInt = #line
-  ) {
-
-    var parser = Parser(source(), swiftVersion: .v5, experimentalFeatures: experimentalFeatures ?? .init())
-
-    let tree = SourceFileSyntax.parse(from: &parser)
-
-    let ifExpr = (tree.statements.first?.item.as(ExpressionStmtSyntax.self)?.expression.as(IfExprSyntax.self))!
-
-    XCTAssertEqual(ifExpr.body.trimmedDescription, body(), file: file, line: line)
-
-  }
-
   func testIfConditions() {
-    assertParse("if true, { }", experimentalFeatures: .trailingComma)
+    assertParse("if true, { }")
 
-    assertParse("if true, { }; { }()", experimentalFeatures: .trailingComma)
+    assertParse("if true, { }; { }()")
 
     assertParse(
       """
       if true, { print("if-body") } else { print("else-body") }
-      """,
-      experimentalFeatures: .trailingComma
+      """
     )
 
     assertParse(
       """
       if true, { print("if-body") } else if true, { print("else-if-body") } { print("else-body") }
-      """,
-      experimentalFeatures: .trailingComma
+      """
     )
 
-    assertParse("if true, { if true { { } } }", experimentalFeatures: .trailingComma)
+    assertParse("if true, { if true { { } } }")
 
-    assertParse("{ if true, { print(0) } }", experimentalFeatures: .trailingComma)
+    assertParse("{ if true, { print(0) } }")
 
-    assertParse("( if true, { print(0) } )", experimentalFeatures: .trailingComma)
+    assertParse("( if true, { print(0) } )")
 
     assertParse(
       """
       if true, { true }() { print(0) }
       """,
       substructure: IfExprSyntax(
         conditions: [
-          ConditionElementSyntax.init(condition: .init(ExprSyntax(stringLiteral: "true")), trailingComma: .commaToken()),
-          ConditionElementSyntax.init(condition: .init(ExprSyntax(stringLiteral: "{ true }()")), trailingComma: nil),
+            ConditionElementSyntax(condition: .expression("true"), trailingComma: .commaToken()),
```

Comment:
Looks like this has some superfluous indentation. Could you run `./swift-syntax-dev-utils format`?